app/styles/app.scss

```diff
@@ -1448,3 +1448,41 @@ nav.branded-navbar {
     margin-left: 15px;
     text-decoration: underline;
 }
+
+.container.pretty-column {
+    .row {
+        padding-left: 20px;
+        .pretty-column-item {
+            padding-left: 20px;
+            &.pretty-column-top {
+                margin-bottom: 30px;
+            }
+        }
+    }
+}
+
+.seeAllSubjects {
+    margin-left: 15px;
+    text-decoration: underline;
+}
+
+.provider-logos {
+	display: flex;
+	flex-wrap: wrap;
+    padding: 20px;
+    max-width: 1000px;
+    margin-left: auto;
+    margin-right: auto;
+    background-image: url(https://staging.osf.io/preprints/assets/img/index-tool-bg.jpg);
```

Comment:
It looks like this line isn't necessary (doesn't resolve; points to staging; looks like it has a background from preprints already). Can you handle that?